docs/content/python/getting-started/quickstart/data/data.md

```diff
@@ -42,24 +41,24 @@ objects: Python objects
 methods: Python methods
 register_classes: |
     ```python
+    class Page:
+      content = "A readable page"
+    
+    class User:
+      role = "guest"
+
     oso.register_class(Page)
```

Comment:
It may be confusing to a reader that we only register `Page` without explaining why.